src/html.pre.js

```diff
@@ -48,9 +49,25 @@ function pre(context) {
   }
 
   // ensure content.data is present
-  if (!context.content.data) {
-    context.content.data = {};
-  }
+  content.data = content.data || {};
+
+  // extract metadata
+  const { meta } = content;
+  // description: text from paragraphs with more than 30 characters and not starting with a link
+  const desc = $sections
```

Comment:
Don't we have something usable here: https://github.com/adobe/helix-pipeline/blob/master/docs/content.md#intro

Reply:
No, unfortunately, this is just the title...

Reply:
I'd still try to generalize the heuristic there.

Reply:
How come jquery here? Should be able to do:

```
document.querySelectorAll('body > div');
```

Reply:
> How come jquery here? 

Mostly because it was already there 😉  but it also does give you some nice chaining...

Reply:
> I'd still try to generalize the heuristic there.

If that means what I think it means: me too 😄  very much open to suggestions, of course.

Reply:
After a quick discussion with @davidnuescheler, I changed the implementation to filter for paragraphs with 10 or more words, and truncate to 25 words. I cheecked a handful of blog articles, and the results looked reasonable.

We can always improve the algorithm later on.